src/libstyles/libstyles.pro

```diff
@@ -2,6 +2,10 @@
 
 QT += core qml quick svg widgets
 
+win32 {
+	DEFINES += LIBSTYLES_LIBRARY
+}
```

Comment:
this is part of the workings of dllexport-dllimport

https://docs.microsoft.com/en-us/cpp/cpp/dllexport-dllimport